polarion_rest_api_client/client.py

```diff
@@ -110,10 +113,14 @@ def __init__(
         batch_size: int = 100,
         page_size: int = 100,
         max_content_size: int = 2 * 1024**2,
+        verify_ssl: truststore.SSLContext | bool | None = None,
```

Comment:
No need to constrain to truststore's subclass here.

```suggestion
        verify_ssl: ssl.SSLContext | bool | None = None,
```